enterprise/server/remote_execution/containers/firecracker/BUILD

```diff
@@ -114,14 +115,16 @@ go_test(
     ],
 )
 
-# TODO(bduffany): once NBD is fully enabled, set NBD flag(s) on
+# TODO(bduffany): once blockio is fully enabled, set blockio flag(s) on
 # firecracker_test and remove this one.
 go_test(
-    name = "firecracker_test_nbd",
+    name = "firecracker_test_blockio",
     timeout = "long",
     srcs = ["firecracker_test.go"],
     args = [
         "--executor.firecracker_enable_nbd=true",
+        # TODO: enable UFFD once it's implemented
+        # "--executor.firecracker_enable_uffd=true",
```

Comment:
I tried enabling this and it triggered a race condition in some of the earlier blockio code that was merged: https://app.buildbuddy.io/invocation/c9ebedf9-9fe2-4e8e-b77d-0095c6165c02?target=%2F%2Fenterprise%2Fserver%2Fremote_execution%2Fcontainers%2Ffirecracker%3Afirecracker_test_blockio